cosmos/providers/dbt/core/operators.py

```diff
@@ -95,7 +97,7 @@ def __init__(
         append_env: bool = False,
         output_encoding: str = "utf-8",
         skip_exit_code: int = 99,
-        python_venv: str = None,
+        dbt_executable_path: str = None,
```

Comment:
do we want a default argument of just `"dbt"` here? what happens if you don't specify a `dbt_executable_path`?

Reply:
In order for it to just work with dbt installed in the Docker image, we should leave it as None.